gcp_variant_transforms/beam_io/bgzf.py

```diff
@@ -99,7 +105,7 @@ def _read_data_from_source(self):
     return self._file.read(self._read_size)
 
   def _is_first_time(self):
```

Comment:
is it really worth wrapping this field in a method?

Reply:
Done.